src/App.css

```diff
@@ -28,6 +28,52 @@
   color: #61dafb;
 }
 
+* {
+  box-sizing: border-box;
+}
+
+.Dropdown-list {
+  list-style-type: none;
+  width: 100%;
+  border: 1px, solid, black;
+  margin: 0;
+  padding: 0;
+}
+
+.Dropdown-list-item {
+  margin: none;
+  border-bottom: 1px solid black;
+}
+
+.Dropdown-list-item:hover {
+  background-color: blue;
+  color: white;
+  cursor: pointer;
+}
+
+.Dropdown-value {
+  width: 100%;
+  margin: auto;
+  padding: 1px 5px;
+}
+
+.Dropdown-menu {
+  margin: auto;
```

Comment:
A few of your css lines are not needed e.g. this `margin: auto' one

A good way to quickly check if a css line is needed is to inspect an element and then you can see the css and tick and un tick css lines and it will automatically update your page, like in the below, let me know if that doesn't make sense as it's super useful.

![image](https://github.com/GuyDav-softwire/bootcamp-react-trainboard/assets/143017308/7ce253f1-9a55-4cd6-b67a-dc3017c66502)